repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -562,7 +566,7 @@ export interface ServiceWithErrors {
 }
 
 export interface ServiceType {
-    id: number;
+    id?: number;
```

Comment:
shouldn't id always be compulsory?

Reply:
its not for this ticket, but this type is used else where so I did not want to remove it completely.